cfgov/paying_for_college/jinja2/paying-for-college/college-cost-blocks/05-federal-loans.html

```diff
@@ -0,0 +1,111 @@
+
+    {{ notification.render(
+        'caution',
+        true,
+        '',
+        'You can borrow less than your school offers and can request more loan funds later if you need to. Only borrow what you need. Entering the amount here does not obligate you to borrow this money.'
+    ) }}
+    <div class="financial-items">
+        <div class="block block__mid">
+            {{
+                input.render({
+                    'label': 'Direct Subsidized Loan',
+                    'id': 'loans__directSub',
+                    'helper_text': 'Cheaper than unsubsidized loans because the government pays your interest until 6 months after you leave school (or drop below half-time)',
+                    'value': '0',
+                    'data_attribute': 'data-financial-item="fedLoan_directSub"'
+                })
+            }}
+            {{
+                text_item.render({
+                    'label': 'Interest rate (%)',
+                    'value': '9.99%',
+                    'data_attribute': 'data-financial-item="rate_directSub"'
+                })
+            }}
+            {{
+                text_item.render({
+                    'label': 'Loan fee (%)',
+                    'value': '9.99%',
+                    'data_attribute': 'data-financial-item="fee_directSub"'
+                })
+            }}
+        </div>
+        <div class="block block__mid">
+            {{
+                input.render({
+                    'label': 'Direct Unsubsidized Loan',
+                    'id': 'loans__directUnsub',
+                    'helper_text': 'You start accruing interest as soon as you receive your first loan',
+                    'value': '0',
+                    'data_attribute': 'data-financial-item="fedLoan_directUnsub"'
+                })
+            }}
+            {{
+                text_item.render({
+                    'label': 'Interest rate (%)',
+                    'value': '9.99%',
+                    'data_attribute': 'data-financial-item="rate_directUnsub"'
+                })
+            }}
+            {{
+                text_item.render({
+                    'label': 'Loan fee (%)',
+                    'value': '9.99%',
+                    'data_attribute': 'data-financial-item="fee_directUnsub"'
+                })
+            }}
+            {{
+                text_item.render({
+                    'label': 'Total federal loans',
+                    'value': '$99, 999',
```

Comment:
I assume this value gets swapped, but is the space in the middle of the number intentional?

Reply:
Nope, typo -- thanks for catching! Updated in https://github.com/cfpb/cfgov-refresh/pull/5742/commits/9237cac212c52dfd381dd65958ae6f2af5f8570c.